docs/developer/app-store/apps/adyen/overview.mdx

```diff
@@ -266,3 +266,34 @@ mutation TransactionProcess($transactionId: ID!, $data: JSON, $customerIpAddress
   }
 }
 ```
+
+## Data processing
+
+The app *stores* and *processes* data that may be PIIs or sensitive:
```

Comment:
Correct me if I'm wrong, but while the app processes PII, it does not store PII.

Reply:
Updated in [`87d749e` (#1415)](https://github.com/saleor/saleor-docs/pull/1415/commits/87d749ec4b8c74491d881efe4c1bbe1d7fc601e5)